Sources/CGFloatExtensions.swift

```diff
@@ -47,5 +47,14 @@ extension CGFloat {
     public static func radiansToDegrees(_ angleInDegrees: CGFloat) -> CGFloat {
         return (180.0 * angleInDegrees) / .pi
     }
-
+    
+    /// EZSE: Returns a random floating point number between 0.0 and 1.0, inclusive.
+    public static func random() -> CGFloat {
```

Comment:
@lfarah suggests better name

As for me, I would stick with random because it's the default behavior.